### DISCUSSION TITLE
Number of simultaneous downloads

### DISCUSSION THREAD

**Discussion by taramisu47** [openaudible/openaudible#1035] [Q&A] [answered]
Is there a setting to change the defaulted 3 simultaneous downloads? Our wifi is fairly slow and 3 at a time causes frequent failures.

**Comment by openaudible** [comment, marked as answer]
Yes, edit the settings.json file. Max is 9 I believe.


Go to prefs, click the folder icon next to Preferences:

(close prefs)

Then edit the settings.json file using a text editor. (making a backup first.) 
```
  "concurrentConversions": 3,
  "concurrentDownloads": 3,
```

Restart OpenAudible.
